arbor/backends/gpu/mechanism_ppack_base.hpp

```diff
@@ -3,30 +3,17 @@
 // Base class for parameter packs for GPU generated kernels:
 // will be included by .cu generated sources.
 
+#include <arbor/mechanism.hpp>
 #include <arbor/fvm_types.hpp>
 
 namespace arb {
 namespace gpu {
 
-// Derived ppack structs may have ion_state_view fields:
-
-struct ion_state_view {
-    using value_type = fvm_value_type;
-    using index_type = fvm_index_type;
-
-    value_type* current_density;
-    value_type* reversal_potential;
-    value_type* internal_concentration;
-    value_type* external_concentration;
-    value_type* ionic_charge;
-};
-
 // Parameter pack base:
-
 struct mechanism_ppack_base {
     using value_type = fvm_value_type;
     using index_type = fvm_index_type;
-    using ion_state_view = ::arb::gpu::ion_state_view;
+    using ion_state_view = ::arb::ion_state_view;
```

Comment:
Is `arb::ion_state_view` is not common between all mechanism descriptions? If so, then we don't need this using statement at all.

Reply:
Yes, that was a mechanical change that is probably redundant.